tests/nlu/selectors/test_selectors.py

```diff
@@ -48,6 +48,7 @@
         ],
     ],
 )
+@pytest.mark.trains_model
```

Comment:
I guess `test_process_gives_diagnostic_data` should also be annotated

Reply:
yeah, these are new tests that keep coming 🤦 
We can get all the stragglers once #7741 is done and merged into this branch.

Reply:
Sounds good! Already forcing people to use `epoch: 1` whereever possible :-D